pages/blog/blog.module.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../styles/globals.scss';
+
+.container {
+	padding: $space-md $space-xxxl;
+	margin-bottom: $space-xxl;
+
+	p {
```

Comment:
the `p` tag appears after the `header` and `h3` in the `jsx` structure so lets move this down in the styles to match its position in the `jsx`